docs/Advanced-Usage/Generating-Different-Targets.rst

```diff
@@ -67,31 +66,32 @@ Specifying A Target Instance
 To generate a specific instance of a target, the build system leverages four Make variables:
 
 1. ``TARGET_PROJECT``: this points the Makefile (`sim/Makefile`) at the right
-   target-specific Makefrag, which defines the generation and MIDAS-level
+   target-specific Makefrag, which defines the generation and Golden Gate-level
```

Comment:
In other places in the docs you left this as MIDAS-level software simulation. Can we leave this as MIDAS-level, since Golden[-]Gate-Level because really long? (and you've left the term Midas hanging around in places like Midas-examples...)

Reply:
I intended to leave ML, ML for now. I agree GG-level isn't good. 

I didn't change anything that would require a code change. The midasexamples will remain the midasexamples for now.